test/spec_mock.rb

```diff
@@ -3,6 +3,12 @@
 require_relative 'helper'
 require 'yaml'
 
+unless YAML.respond_to?(:unsafe_load)
```

Comment:
Can you add a note to explain why this is added and when it can be removed?

Reply:
Done.